data/templates/ssh/sshd_config.j2

```diff
@@ -96,3 +96,7 @@ DenyGroups {{ access_control.deny.group | join(' ') }}
 # sshd(8) will send a message through the encrypted channel to request a response from the client
 ClientAliveInterval {{ client_keepalive_interval }}
 {% endif %}
+
+{% if rekey.data is vyos_defined and rekey.time is vyos_defined %}
```

Comment:
As both options are required to set this in the ssh configuration file you should probably add a `verify()` step. Else the user might not know why the config item did not appear.

Reply:
Added check for data limit and as an option time limit